### PR TITLE
Make direct upload default strategy for deploy

### PR DESCRIPTION
**Pull request by piotrze**
Fixes: https://github.com/mdyd-dev/desksnearme/issues/8860

### CODE REVIEW

**Comment by pavelloz** [comment]
This way it wont be merged 👍

**Comment by piotrze** [comment]
@pavelloz :full_moon:

File: bin/pos-cli-deploy.js [hunk 1]
```diff
@@ -33,7 +34,7 @@ program
       CI: process.env.CI === 'true',
       // TODO: Get rid off global system env, make it normal argument to function.
       PARTIAL_DEPLOY: !!params.partialDeploy,
-      DIRECT_ASSETS_UPLOAD: params.directAssetsUpload
```
Comment (pavelloz) [inline bin/pos-cli-deploy.js:36]: test

File: lib/assets.js [hunk 2]
```diff
@@ -37,17 +36,12 @@ const deployAssets = async gateway => {
...
-    ora({
-      text: 'Generating and uploading new assets manifest...',
-      stream: process.stdout,
-      spinner: 'bouncingBar'
-    }).start();
     const manifest = await manifestGenerate();
     logger.Debug(manifest);
     await gateway.sendManifest(manifest);
-    logger.Success('Deploy finished.');
+    logger.Debug('Uploading assets');
```
Comment (pavelloz) [inline lib/assets.js:44]:
```suggestion
test 1
```

File: lib/deploy/defaultStrategy.js [hunk 3]
```diff
@@ -10,13 +11,22 @@ const createArchive = async(env) => {
 const uploadArchive = async(env) => {
   const res = await push.push(env);
   if (!res) throw 'Server did not accept release file.';
+
```
Comment (pavelloz) [inline lib/deploy/defaultStrategy.js:14]:
```suggestion

```

**Comment by pavelloz** [comment]
holy moly

**Pull request merged**